libc/src/__support/FPUtil/generic/FMA.h

```diff
@@ -231,6 +231,8 @@ fma(InType x, InType y, InType z) {
   OutStorageType result = 0;
   int r_exp = 0; // Unbiased exponent of the result
 
+  int round_mode = fputil::quick_get_round();
```

Comment:
Put this in the `else` clause if it's only used in there.

Reply:
It's also used after the `if`/`else`. I moved the `round_mode` declaration from there to avoid calling `quick_get_round()` twice.